draft-ietf-rats-corim.md

```diff
@@ -2607,6 +2618,28 @@ If no entry is found, the comparison MUST return false.
 Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}} after equivalence has been found according to {{sec-comid-integrity-registers}}.
 Note that it is not required for all the entries in the candidate entry to be used during matching: the condition ECT could consist of a subset of the device's register space. In TPM parlance, a TPM "quote" may report all PCRs in Evidence, while a condition ECT could describe a subset of PCRs.
 
+##### Comparison for raw-integral entries
+
+The ACS entry value stored under `measurement-values-map` codepoint 15 is a raw integral value, which must have type `raw-integral-type-choice`.
+
+Consider an `integral` ACS entry value named ENTRY in a `measurement-values-map` codepoint (e.g., 15) that allows comparing `integral` against a either another `integral` or an `integral-range` named CONDITION.
+
+*  If CONDITION is an `integral` then an equality comparison is performed with ENTRY. Equality between major type 0 and unsigned bignum or conversely major type 1 and negative bignum first requires translation of small ints to bignum to compare.
+
+*  If CONDITION is an `integral-range` tagged with #6.564, then a range inclusion comparison is performed.
+The comparison MUST return true if and only if all the following conditions are true:
+    + CONDITION.min is `null` or ENTRY is greater than or equal to the CONDITION.min.
+    + CONDITION.max is `null` or ENTRY is less than or equal to CONDITION.max.
+
+Consider an `integral-range` or `integral-range` tagged with #6.564 value named ENTRY in a `measurement-values-map` codepoint (e.g., 15) that allows comparing an `int-range` against either another `int-range` or an `int` named CONDITION.
```

Comment:
```suggestion
Consider an `integral` or `integral-range` (CBOR tag 564) value named ENTRY in a `measurement-values-map` codepoint (i.e., 15) that allows comparing an `integral-range` against either another `integral-range` or an `integral` named CONDITION.
```

Reply:
Is the justification for changing 'int' to 'integral' because the numeric is defined in terms of int, uint, and big integers?

Reply:
Will remove bigint and just say raw-int for simplicity.